### PR TITLE
Update for cycle 18

### PR DESCRIPTION
**Pull request by jeanconn**
Update for cycle 18

### CODE REVIEW

File: make_reports.py [hunk 1]
```diff
@@ -59,9 +59,8 @@ def get_options():
 WHERE
 ((t.status='unobserved' OR t.status='partially observed' OR t.status='untriggered' OR t.status='scheduled')
 AND NOT(t.ra = 0 AND t.dec = 0)
-AND NOT(t.ra IS NULL OR t.dec IS NULL)
-AND (t.obs_ao_str <= '{}'))
-ORDER BY t.obsid""".format(CYCLE)
```
Comment (jeanconn) [inline make_reports.py:64]: I don't see a reason to not include all outstanding targets.

**Comment by jeanconn** [comment]
Review output at 

http://cxc.cfa.harvard.edu/mta/ASPECT/aca_lts_eval_precheck_18/

**Comment by jeanconn** [comment]
@taldcroft Are you OK with these changes?

**Comment by taldcroft** [comment]
Looks good to me.

**Pull request merged**